opentile/common.py

```diff
@@ -166,9 +166,8 @@ def tile_size(self) -> Size:
     def tiled_size(self) -> Size:
         """The size of the image when tiled."""
         if self.tile_size != Size(0, 0):
-            return self.image_size / self.tile_size
-        else:
-            return Size(1, 1)
+            return self.image_size.ceil_div(self.tile_size)
```

Comment:
It feels like ceil_div is the main path of the code, and return Size(1,1) is a special case, so I would flip the if clause and explain why a zero size is counted as 1:

```python
if self.tile_size == Size(0, 0):
    # Explain the reason for this..
    # ...
    return Size(1, 1)

return self.image_size.ceil_div(self.tile_size)
```